code/modules/admin/verbs/randomverbs.dm

```diff
@@ -47,6 +47,52 @@
 	admin_ticket_log(M, msg)
 	SSblackbox.record_feedback("tally", "admin_verb", 1, "Subtle Message") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/client/proc/cmd_admin_mod_antag_rep(client/C in GLOB.clients, var/operation)

```

Comment:
This should probably be admin holder function

Reply:
Indeed